pkg/aws/eni/node_manager_test.go

```diff
@@ -70,13 +70,13 @@ func (e *ENISuite) TestGetNodeNames(c *check.C) {
 	c.Assert(mngr, check.Not(check.IsNil))
 
 	node1 := newCiliumNode("node1")
-	mngr.Update(newCiliumNode("node1"))
+	mngr.Upsert(node1)
```

Comment:
nit: Let's remove the previous line if you do not need the variable.
```suggestion
	mngr.Upsert(newCiliumNode("node1"))
```